content/courses/native-onchain-development/serialize-instruction-data-frontend.md

```diff
@@ -281,7 +294,7 @@ The public key of the Solana program we’ll use for this application is
 #### 1. Download the starter code
 
 Before we get started, go ahead and download the
-[starter code](https://github.com/Unboxed-Software/solana-movie-frontend/tree/starter).
+[starter code](https://github.com/EmekaManuel/movie-review-dapp/tree/starter).
```

Comment:
Update this URL to point tothe solana-developers fork.

Reply:
> Update this URL to point tothe solana-developers fork.

updated ✅